vendor-extra/ContentPageBundle/tests/GuzzleTestCase.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentPageBundle;
+
+use Csa\GuzzleHttp\Middleware\Cache\Adapter\MockStorageAdapter;
+use Csa\GuzzleHttp\Middleware\Cache\Adapter\StorageAdapterInterface;
+use Csa\GuzzleHttp\Middleware\Cache\MockMiddleware;
+use GuzzleHttp\Client;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use VirtualFileSystem\FileSystem;
+
+trait GuzzleTestCase
+{
+    /** @var ClientInterface */
+    protected $client;
+    /** @var FileSystem */
+    protected $filesystem;
+    /** @var StorageAdapterInterface */
+    protected $mock;
+
+    /**
+     * @before
+     */
+    public function setUpGuzzle() : void
+    {
+        $this->filesystem = new FileSystem();
+
+        $stack = HandlerStack::create(new MockHandler());
+        $stack->push(
+            new MockMiddleware(
+                $this->mock = new MockStorageAdapter(
+                    $this->filesystem->path(''),
```

Comment:
Easier than https://github.com/elifesciences/api-sdk-php/blob/master/test/InMemoryStorageAdapter.php.

Reply:
It seems easier to setup. What is the empty path?

Reply:
To get the absolute root path. Might be a more obvious way...